codec/optional_packer.go

```diff
@@ -4,62 +4,94 @@
 package codec
 
 import (
+	"math"
+
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/hypersdk/consts"
 	"github.com/ava-labs/hypersdk/crypto"
 )
 
-// Inspired by: https://yourbasic.org/golang/bitmask-flag-set-clear
-type bits uint8
+// MaxItems stores the maximum items an optional packer can hold.
+// Since a bitset can range from [0, INT_MAX] the max number of
+// flags in a bitset is log2(INT_MAX).
+var MaxItems = uint8(math.Log2(float64(consts.MaxInt)))
```

Comment:
Let's make this a much smaller number. How about 64?

Reply:
I'd also be ok to remove this entirely now

Reply:
MaxItems is actually 63 in this case